--- ./t4p4s.sh
+++ ./t4p4s.sh
@@ -1017,7 +1017,7 @@ if [ "$(optvalue c)" != off ]; then
     cd - >/dev/null

     cd ${T4P4S_TARGET_DIR}
-    sudo CC="ccache $T4P4S_CC" CC_LD="$T4P4S_LD" CFLAGS="$FLTO" $MESON_CMD $MESON_OPTS -Dbuildtype=$MESON_BUILDTYPE build >$T4P4S_LOG_DIR/20_meson.txt 2>&1
+    sudo CC="ccache $T4P4S_CC" CC_LD="$T4P4S_LD" CFLAGS="-lm $FLTO" $MESON_CMD $MESON_OPTS -Dbuildtype=$MESON_BUILDTYPE build >$T4P4S_LOG_DIR/20_meson.txt 2>&1
     MESON_ERRCODE="$?"
     [ "$MESON_ERRCODE" != "0" ] && ERRCODE=$ERRCODE_C
     exit_on_error "$ERRCODE" "Meson invocation $(cc 2)failed$nn with error code $(cc 3)${MESON_ERRCODE}$nn, see $(cc 1)$(realpath --relative-to="$STARTPWD" $T4P4S_LOG_DIR/20_meson.txt)$nn"
